README.html

```diff
@@ -0,0 +1,1028 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><style>body {
```

Comment:
What is the purpose of this file? Is it just a copy of `README.md` with the addition of a large custom style sheet?